connector_ecommerce/components/sale_order_onchange.py

```diff
@@ -77,7 +85,7 @@ def play(self, order, order_lines):
         :rtype: dict
         """
         # play onchange on sales order
-        order = self.play_onchanges("sale.order", order, self.order_onchange_fields)
+        order = self.play_onchanges("magento.sale.order", order, self.order_onchange_fields)
```

Comment:
I suppose his change is not necessary any more?

Reply:
Absolutely, removed it.